wger/gym/templates/gym/member_list.html

```diff
@@ -323,4 +347,23 @@ <h4>{% trans "Emails" %}</h4>
         {% trans "Add member" %}
     </a>
 {% endif %}
+<script>
+function openCity(evt, cityName) {
+    var i, tabcontent, tablinks;
+    tabcontent = document.getElementsByClassName("tabcontent");
+    for (i = 0; i < tabcontent.length; i++) {
+        tabcontent[i].style.display = "none";
+    }
+    tablinks = document.getElementsByClassName("tablinks");
+    for (i = 0; i < tablinks.length; i++) {
+        tablinks[i].className = tablinks[i].className.replace(" active", "");
+    }
+    document.getElementById(cityName).style.display = "block";
+    evt.currentTarget.className += " active";
+}
+
+// Get the element with id="defaultOpen" and click on it
+document.getElementById("defaultOpen").click();
+</script>
+
```

Comment:
here too

Reply:
I was unable to move the Jquery for this particular functionality into a different file